src/index.ts

```diff
@@ -12,7 +11,13 @@ const hook: Hook.CommandNotFound = async function (opts) {
 
   if (commandIDs.length === 0) return
   function closest(cmd: string): string {
-    return _.minBy(commandIDs, c => Levenshtein.get(cmd, c))!
+    // we'll use this array to keep track of which key is the closest to the users entered value.
+    // keys closer to the index 0 will be a closer guess than keys indexed further from 0
+    // an entry at 0 would be a direct match, an entry at 1 would be a single character off, etc.
+    const index: string[] = []
+    // eslint-disable-next-line no-return-assign
+    commandIDs.map(id => (index[Levenshtein.get(cmd, id)] = id))
```

Comment:
in SDR, we've used `.get(a,b, { useCollator: true})` to make the algo consider case-insensitivity to be closer than "just any one-letter difference".  I don't think our CLI cares because we're all lowercase, but other oclif users might care about it.